__tests__/utils/search.js

```diff
@@ -6,22 +6,34 @@ describe('search', () => {
   it('should seach for packages', async () => {
     const mockHttpResponse = {
       results: [
-        { name: ['foobar-default-theme'] },
-        { name: ['foobar-default-plugin'] },
+        {
+          name: ['foobar-default-theme'],
+          description: ['Foobar default theme'],
+        },
+        {
+          name: ['foobar-default-plugin'],
+          description: ['Foobar default plugin'],
+        },
       ],
     };
     require('http').__setMockResponse(JSON.stringify(mockHttpResponse));
 
-    const pkg = 'foobar';
+    const q = 'foobar';
     const mockResponse = [
-      { name: ['foobar-default-theme'] },
-      { name: ['foobar-default-plugin'] },
+      {
+        name: 'foobar-default-theme',
+        desc: 'Foobar default theme',
+      },
+      {
+        name: 'foobar-default-plugin',
+        desc: 'Foobar default plugin',
+      },
     ];
-    expect(await search.searchPackages(pkg)).toEqual(mockResponse);
+    expect(await search.searchPackages(q)).toEqual(mockResponse);
   });
 
   it('should retrieve the package url', () => {
     const pkg = 'foobar';
-    expect(search.getSearchUrl(pkg)).toEqual('http://npmsearch.com/query?q=foobar%20AND%20(keywords:dext-theme%20OR%20keywords:dext-plugin)&fields=name');
+    expect(search.getSearchUrl(pkg)).toEqual('https://npmsearch.com/query?q=foobar%20AND%20(keywords:dext-theme%20OR%20keywords:dext-plugin)&fields=name,description');
```

Comment:
This doesn't work on my machine for some reason...

`Error: Protocol "https:" not supported. Expected "http:"
    at new ClientRequest (_http_client.js:55:11)
    at Object.exports.request (http.js:31:10)
    at Object.exports.get (http.js:35:21)
    at Promise (/Users/wolfwood/dev/dext-cli/node_modules/dext-core-utils/src/utils/search.js:23:15)
    at searchPackages (/Users/wolfwood/dev/dext-cli/node_modules/dext-core-utils/src/utils/search.js:19:29)
    at Promise (/Users/wolfwood/dev/dext-cli/node_modules/dext-core-utils/src/api/index.js:29:3)
    at Object.search (/Users/wolfwood/dev/dext-cli/node_modules/dext-core-utils/src/api/index.js:28:30)
    at Args.args.command (/Users/wolfwood/dev/dext-cli/cli.js:12:14)
    at Args.runCommand (/Users/wolfwood/dev/dext-cli/node_modules/args/dist/index.js:327:37)
    at Args.parse (/Users/wolfwood/dev/dext-cli/node_modules/args/dist/index.js:429:12)`